enforcers/kube_enforcer/kubernetes_and_openshift/manifests/kube_enforcer/001_kube_enforcer_config.yaml

```diff
@@ -103,20 +103,26 @@ kind: ClusterRole
 metadata:
   name: aqua-kube-enforcer
 rules:
-  - apiGroups: ["*"]
-    resources: ["pods", "nodes", "namespaces", "deployments", "jobs", "cronjobs", "daemonsets", "replicasets", "replicationcontrollers", "statefulsets", "clusterroles", "clusterrolebindings", "componentstatuses", "services" ]
+  - apiGroups: [""]
+    resources: ["pods", "nodes", "namespaces", "replicationcontrollers", "componentstatuses", "services" ]
     verbs: ["get", "list", "watch"]
-  - apiGroups: [ "*" ]
+  - apiGroups: [ "apps" ]
+    resources: [ "deployments", "daemonsets" ,"replicasets","statefulsets" ]
```

Comment:
comma and space are not aligned properly.
`"deployments", "daemonsets" ,"replicasets","statefulsets" `
it should be 
`"deployments", "daemonsets", "replicasets", "statefulsets" `

It may not impact anywhere, but good for formating